views/admin/index.php

```diff
@@ -22,23 +22,21 @@
 
 ?>
 
-
 <?php $this->beginContent('@legal/views/admin/layout.php') ?>
 <div class="panel-body">
 
-
     <?php $form = ActiveForm::begin(['id' => 'configure-form', 'enableClientValidation' => false, 'enableClientScript' => false]); ?>
 
     <?= $form->field($model, 'enabledPages')->checkboxList($enabledPages); ?>
     <?= $form->field($model, 'defaultLanguage')->dropDownList(Yii::$app->i18n->getAllowedLanguages(), ['data-ui-select2' => '']); ?>
 
     <?= $form->field($model, 'showAgeCheck')->checkbox(); ?>
+    <?= $form->field($model, 'defaultAge')->textInput(); ?>
```

Comment:
I'm thinking of changing this to the below so that there is a hint to tell users to enter a number value :thinking:

`<?= $form->field($model, 'defaultAge')->textInput()->hint(\Yii::t('LegalModule.base', 'Please enter a number value.')); ?>`

Reply:
Just don't want to recompile the translations again :sweat_smile: